### PR TITLE
Feature user profiles

### PR DESCRIPTION
**Pull request by hosseintoussi**

### CODE REVIEW

**Comment by aaronmyatt** [comment]
Damn this hound is noisy

@hosseintoussi here's the dirty implementation of user profiles. I didn't do more since you say you want to play around with it.

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+  def index
+    @users = User.all
+  end
+
+  def show
+    @cards = current_user.cards.order('created_at')
+    @total_cards = @cards.size
+    @latest_card = @cards.last
+    @liked = current_user.find_voted_items.sort_by { |like| like.created_at }
```
Comment (hosseintoussi) [inline app/controllers/users_controller.rb]: 👍 hound

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
+class UsersController < ApplicationController
+  before_action :set_user, only: [:show, :edit, :update]
+
+  def index
+    @users = User.all
+  end
+
+  def show
+    @cards = current_user.cards.order('created_at')
+    @total_cards = @cards.size
```
Comment (hosseintoussi) [inline app/controllers/users_controller.rb]: why not count?

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (hosseintoussi) [inline app/controllers/users_controller.rb]: So I was thinking about having this much of instance variables and passing them all into the views where we could just call those methods on the variables in view. 
did some googling against instance variables and learned something new. Have a look at this answer, it makes a lot of sense to me http://stackoverflow.com/questions/2503838/rails-should-partials-be-aware-of-instance-variables#answer-13752155

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (aaronmyatt) [inline app/controllers/users_controller.rb]: These guys: https://github.com/bbatsov/ruby-style-guide - suggest size over count

Comment (hosseintoussi) [inline app/controllers/users_controller.rb]: 👍  we should come up with our own style guide :D theres too many of them 💃

**Comment by hosseintoussi** [comment]
can we remove the png file? I think we are not using it. Also there are two other files one coffee and one scss that were generated by rails. i think we can delete both of them too.

**Comment by aaronmyatt** [comment]
Roger

File: app/assets/javascripts/tabular.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+$( document ).on("page:change", function () {
+  $('.menu .item').tab();
```
Comment (hosseintoussi) [inline app/assets/javascripts/tabular.js]: 👍  lets change the quotes here too

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -0,0 +1,30 @@
+class UsersController < ApplicationController
+  before_action :set_user, only: [:show, :edit, :update]
+
+  def index
+    @users = User.all
+  end
+
+  def show
+    @cards = current_user.cards.order("created_at")
+    @liked = current_user.find_voted_items.sort_by(&:created_at)
```
Comment (hosseintoussi) [inline app/controllers/users_controller.rb]: as you are moving these to models, perhaps you can put a limit on them too :) 🙇

File: app/views/users/_user_detail_card.html.erb [hunk 5]
```diff
@@ -0,0 +1,23 @@
+<div class="card">
+
+    <div class="image">
+      <%= image_tag('https://www.drupal.org/files/profile_default.png') %>
+    </div>
+    <div class="content">
```
Comment (hosseintoussi) [inline app/views/users/_user_detail_card.html.erb]: Where can we display user's name here?

**Pull request merged**